libs/activities/pom.xml

```diff
@@ -36,11 +36,6 @@
                          was the maven release plugin was not committing during the release.
                      -->
                     <version>2.4.1</version>
```

Comment:
Off the top of my head, I would agree with removing the old version. This may be a bit residual from when these were separate repositories. Also some of the issue were conflicts between specific versions of git and the maven release plugin, which I believe we are long past now.